docs/section22/Rule22-42.md

```diff
@@ -0,0 +1,98 @@
+            - create a list of the expected temperatures for the chilled water temperature for the validation calculations - expected_chwt_temps - units are (F).  The values are selected to capture the minimum, maximum, rating condition and some points in between: `expected_cwt_temps = [39,45,50,55]`
+            - create a list of the expected entering condenser temperatures for the condenser water temperature for the validation calculations - expected_ect_temps - units are (F).  The values are selected to capture the minimum, maximum, rating condition and some points in between: `expected_ect_temps = [60,104,85,72.5,97.5]`
+
+            - get the coefficients for the EIR-f-T curve (IP units) using a table lookup for table J-6.  This assumes that the lookup function will return a list with the coefficients in order (C1,C2,...): `eir_f_t_coefficients = table_J_2_lookup(curve_set,"EIR-f-T")`
+            - get the coefficients for the CAP-f-T curve (IP units) using a table lookup for table J-6.  This assumes that the lookup function will return a list with the coefficients in order (C1,C2,...): `cap_f_t_coefficients = table_J_2_lookup(curve_set,"Cap-f-T")`
+            - get the coefficients for the PLR curve (IP units) using a table lookup for table J-6.  This assumes that the lookup function will return a list with the coefficients in order (C1,C2,...): `plr_coefficients = table_J_2_lookup(curve_set,"EIR-f-PLR")`
+
+            - create a variable (capacity_validation_status) to track whether the capacity validation passes and set it to "PASS" - if the capacity validation fails at any point, this variable will change to give the values at which the capacity validation failed: `capacity_validation_status = "PASS"`
+            - create a variable (power_validation_status) to track whether the power validation passes and set it to "PASS" - if the power validation fails at any point, this variable will change to give the values at which teh power validation failed: `power_validation_status = "PASS"
+
+            - convert the chiller.capacity_validation_points to a dictionary with the keys being a list of two values - [chilled_water_supply_temperature, condenser_temperature], and values being the capacity_validation_point itself.  Sorting the validation points will allow the calculations that follow to be much simpler.  Start by creating a dict capacity_validation_pts_dict: `capacity_validation_pts_dict = {}`
+            - look at each value in chiller.capacity_validation_points: `for capacity_validation_point in chiller.capacity_validation_points:`
+                - add the point to capacity_validation_pts_dict: `capacity_validation_pts_dict[[capacity_validation_point.chilled_water_supply_temperature, capacity_validation_point.condenser_temperature]] = capacity_validation_point`
+            
```

Comment:
so this is "capacity_validation_pts_dict[[capacity_validation_point.chilled_water_supply_temperature, capacity_validation_point.condenser_temperature]] = capacity_validation_point.result" ?

Reply:
yes, will change it to this.  Makes it a bit simpler

Reply:
@KarenWGard I think this syntax needs to be revised. I think right now you have two lists rather than a key in the dictionary. I could be missing something.

Reply:
updated